### PR TITLE
[#2169] Improve mail-template styling

### PR DESCRIPTION
**Pull request by jiromaykin**
detailed issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/2169

overall issue: https://taiga.maykinmedia.nl/project/open-inwoner/task/2134

These mostly concern `case_document_notification` ,` case_status_notification_action_required` and `case_status_notification`.

ToDo

- [x] - add a little bit more placeholder-data so CKeditor preview/Testmails look nicer
- [x] - Spacing between 'met vriendelijke groet…' and 'dit bericht is automatisch…' 2X 
- [x] - faulty whitespace in white rounded-border area
- [x] - add sharper/higher rez icons
- [x] - check if there is extra whitespace in the empty columns (making the wrapping Table responsive, max width 600px)
- [x] - correct the blue info hex color
- [x] - Spacing 'login' + "U hoeft niets te doen" needs to be larger (info notificatie)   
- [x] - "Log in" should not be underlined (info notificatie)
- [x] - Cell-widths tablecells for phonenumber and contactlink should be reversed  (2X)

EXTRA

- [x] align-right for 1 right-column + align left on mobile
- [x] small CKEditor style improvements

### CODE REVIEW

File: src/open_inwoner/templates/mail/_base.html [hunk 1]
```diff
@@ -131,6 +137,7 @@
 
     .button--primary {
         background-color: {{ theming.primary_color }};
+        border: 1px solid{{ theming.primary_color }};
```
Comment (Bartvaderkin) [inline src/open_inwoner/templates/mail/_base.html:140]: There should be a space between `solid` and the `{{`

**Review by Bartvaderkin — CHANGES_REQUESTED** [review]

File: src/open_inwoner/templates/mail/_base.html [hunk 2]
```diff
@@ -113,6 +118,7 @@
         --color-accent: {{ theming.accent_color }};
         --color-primary: {{ theming.primary_color }};
         --color-secondary: {{ theming.secondary_color }};
+        margin: 0;
```
[the remark this answers is not in the document]
Comment (jiromaykin) [reply src/open_inwoner/templates/mail/_base.html:121]: We need this margin on the base template Body in order to get rid of default spacing ~ which in the issue from Groningen was mentioned as "Gele/blauwe vlak zou over hele breedte van het scherm, is nu als deel" which was a bit weirdly worded but this margin was causing it.

**Review by pi-sigma — CHANGES_REQUESTED** [review]
Template language should be consistent

File: src/open_inwoner/conf/parts/expiring_plan.html [hunk 3]
```diff
@@ -17,5 +17,5 @@
 </table>
 
 <p><a href="{{ plan_list_link }}">Go to your plans</a></p>
```
[the remark this answers is not in the document]
Comment (jiromaykin) [reply src/open_inwoner/conf/parts/expiring_plan.html:19]: Probably, but this PR for #2169 is really only about styling and about the 3 case_status templates; I'd rather change all the 'other templates properly in task #2120 -  I just wanted to do a small correction for the paragraphs here.

**Review by pi-sigma — APPROVED** [review]

**Review by Bartvaderkin — APPROVED** [review]

**Comment by alextreme** [comment]
Mag gemerged worden na verhelpen conflict

**Pull request merged**